main/src/ca/uwaterloo/flix/language/phase/Lexer.scala

```diff
@@ -345,8 +343,8 @@ object Lexer {
           TokenKind.Dot
         }
       case '$' if peek().isUpper => acceptBuiltIn()
-      case '₹' =>
-        // Don't include the rupee sign in the name
+      case '$' if peek().isLower =>
```

Comment:
Does this break if a Java name has a `$` in it?

Reply:
Thats my question: https://github.com/flix/flix/issues/9213